lab_1_classify_by_unigrams/main.py

```diff
@@ -20,6 +27,19 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
     :return: a dictionary with frequencies
     """
 
+    if not isinstance(tokens, list):
+        return None
+
+    dictionary = {}
+    token_quantity = len(tokens)
+    for i in tokens:
+        if not isinstance(i, str):
+            return None
+        if i not in dictionary:
+            dictionary[i] = tokens.count(i) / token_quantity
+            pass
```

Comment:
For what is this pass?

Reply:
this pass is only for pointing it while debug as it gives the result dictionary